text/0060-refashion-updates-apis.md

```diff
@@ -102,9 +102,9 @@ The motivation is to stabilize the current `update` resource to a version that c
 |  -         | indexCreation |
 |  -         | indexUpdate   |
 |  -         | indexDeletion |
-| DocumentsAddition | documentsAddition  |
-| DocumentsPartial | documentsPartial  |
-| DocumentsDeletion  | documentsDeletion |
+| DocumentsAddition | documentAddition  |
+| DocumentsPartial | documentPartial  |
```

Comment:
Sorry, I'm "relou", but is `document partial` means something? What do you think @dichotommy?

Reply:
Doesn't mean a lot to me. I'm guessing it's... a document update? As in, `PUT /indexes/:index_uid/documents`

Reply:
It's a deeper problem than that, we didn't find a better term during our thinking. A document addition can also be an update. Partial here is only to indicate that the update is done from the `PUT` endpoint. @MarinPostma correct me if I'm wrong.

Reply:
I think I understand what the idea was with the name (partial update of document), but since users are required to provide the entire document anyway (it's `PUT` not `PATCH`), and this endpoint can also be used to add entirely new documents, it seems like not the best choice.

Reply:
If I've correctly understood which endpoint this task type represents, why not `documentUpdate`?

Reply:
I think we should use the verb `PATCH` but in this case, I'm not sure because we don't process only one resource but several `documents` resources. It can make sense if we say that we treat an index resource and that we PATCH (partially) a part of the documents within the index resource.

Reply:
That's something we'll deal with later. We could have a single method that allows you to add documents, update and delete documents from a single payload. (This is just an idea for now)